frontend/src/models/propertyDefinitionsModel.ts

```diff
@@ -96,6 +96,12 @@ export const propertyDefinitionsModel = kea<
             (s) => [s.propertyStorage],
             (propertyStorage): PropertyDefinition[] => propertyStorage.results || [],
         ],
+        findProperty: [
```

Comment:
Suggestion: `findPropertyDefinitionByName` to make it really clear what we're doing here. During the first pass over the changes I wasn't sure what this function was supposed to do in `propertyFilterLogic`.